opengrok-indexer/src/main/java/org/opengrok/indexer/history/RazorRepository.java

```diff
@@ -256,10 +256,6 @@ InputStream getHistoryGet(String parent, String basename, String rev) {
     Annotation annotate(File file, String revision)
             throws IOException {
         // @TODO : Rename & Delete Support
-        File rcsFile = getRazorArchiveRCSFileFor(file);
-        if (rcsFile != null && rcsFile.exists()) {
-            return RCSRepository.annotate(file, revision, rcsFile);
-        }
```

Comment:
why is this no longer needed?

Reply:
It should work as long as Razor automatically creates the `RCS/XYZ.txt,v` file.